signer/tests/integration/contracts.rs

```diff
@@ -205,40 +201,7 @@ async fn complete_deposit_wrapper_tx_accepted<T: AsContractCall>(contract: Contr
     tokio::time::sleep(std::time::Duration::from_secs(5)).await;
 
     let txid = signer.sign_and_submit(&contract).await;
-
-    // The submitted transaction tends to linger in the mempool for quite
-    // some time before being confirmed in a Nakamoto block (best guess is
-    // 5-10 minutes). It's not clear why this is the case.
-    if true {
-        println!("{}", txid);
-        return;
-    }
-
-    let client = stacks_client();
-
-    // We need a block id
-    let info = client.get_tenure_info().await.unwrap();
-    let storage = Store::new_shared();
-
-    let tenures = stacks::api::fetch_unknown_ancestors(&client, &storage, info.tip_block_id)
-        .await
-        .unwrap();
-
-    let blocks = tenures
-        .into_iter()
-        .flat_map(|tenure| tenure.into_blocks())
-        .collect::<Vec<_>>();
-
-    let transactions = postgres::extract_relevant_transactions(&blocks, &signer.wallet.address());
-    assert!(!transactions.is_empty());
-
-    let txids = transactions
-        .iter()
-        .map(|stx| blockstack_lib::burnchains::Txid::from_bytes(&stx.txid))
-        .collect::<Option<HashSet<_>>>()
-        .unwrap();
-
-    assert!(txids.contains(&txid));
+    println!("{txid}");
```

Comment:
Do we need this print?

Reply:
As I see this test does not have nor `assert`'s nor `unwrap`'s so we can delete it (or add some assertions)

Reply:
Well, I suppose we could delete it, since it is ignored already. However, I actually get some use out of these tests every now and again, and the print is useful when I use it.

This test is supposed to check that the stacks node will accept our contract call transactions and that they get confirmed. The call to `sign_and_submit` will panic if the transaction gets rejected, so it tests acceptance. The part that is deleted is supposed to check that the transaction gets confirmed. It never did that "correctly", it needs to wait for some arbitrary amount of time for the confirmation, so we'd need to change it when we finally run a stacks node in CI. Oh, and we should delete the print when the test is enabled.

Reply:
Oookay, makes sense